packages/rocketchat-ui/lib/chatMessages.coffee

```diff
@@ -39,9 +24,60 @@ class @ChatMessages
 			index++
 		return -1
 
+	recordInputAsDraft: () ->
+		console.log(this.editing)
+		id = this.editing.id
+		record = this.records[id] || {}
+		draft = this.input.value
+
+		if(draft is record.original)
+			this.clearCurrentDraft()
+		else
+			record.draft = draft
+			this.records[id] = record
+
+	recordOriginalMessage: (message) ->
+		record = this.records[message._id] || {}
+		record.original = message.msg
+		this.records[message._id] = record
+
+	getMessageDraft: (id) -> return this.records[id]
+	clearMessageDraft: (id) -> delete this.records[id]
```

Comment:
whatever you prefer =)

On 05/02/2016 08:10 PM, Rodrigo Nascimento wrote:
>
> In packages/rocketchat-ui/lib/chatMessages.coffee 
> <https://github.com/RocketChat/Rocket.Chat/pull/3102#discussion_r61768201>:
>
> > +		record = this.records[id] || {}
> > +		draft = this.input.value
> > +
> > +		if(draft is record.original)
> > +			this.clearCurrentDraft()
> > +		else
> > +			record.draft = draft
> > +			this.records[id] = record
> > +
> > +	recordOriginalMessage: (message) ->
> > +		record = this.records[message._id] || {}
> > +		record.original = message.msg
> > +		this.records[message._id] = record
> > +
> > +	getMessageDraft: (id) -> return this.records[id]
> > +	clearMessageDraft: (id) -> delete this.records[id]
>
> Can we break these in 2 lines each? Just to keep code style?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/RocketChat/Rocket.Chat/pull/3102/files/a830464fedddaaa867d7e6d68dc0f77b38f1ae89#r61768201>
>